Tasks/AppCenterDistributeV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -38,5 +38,6 @@
   "loc.messages.CannotFindAnyFile": "Cannot find any file based on %s.",
   "loc.messages.FoundMultipleFiles": "Found multiple files matching %s.",
   "loc.messages.FailedToCreateFile": "Failed to create %s with error: %s.",
-  "loc.messages.FailedToFindFile": "Failed to find %s at %s."
+  "loc.messages.FailedToFindFile": "Failed to find %s at %s.",
+  "loc.messages.DeprecatedTask": "The AppCenterDistribute@1 (App Center distribute) task has been deprecated since November 14, 2022 and will soon be retired. Use the AppCenterDistribute@3 task instead."
```

Comment:
This overrides the deprecationMessage in task.json?

Reply:
The deprecationMessage is not overriden, currently in order to read a string in a Task that string should be present inside "messages" property of task.json that's why I had to duplicate the deprecationMessage inside "messages", there can be future work to read the strings outside of "messages".